cropduster/templatetags/cropduster_tags.py

```diff
@@ -59,6 +60,14 @@ def get_crop(image, crop_name, exact_size=False, **kwargs):
     data = {}
     data['url'] = getattr(Image.get_file_for_size(image, crop_name), 'url', None)
 
+    if image.related_object:
```

Comment:
Why not move this stuff up here? I think we probably want to deprecate `exact_size`, but that should probably be a separate commit, no?